bindings/go/oci/repository.go

```diff
@@ -38,6 +38,15 @@ import (
 	"ocm.software/open-component-model/bindings/go/runtime"
 )
 
+// plusSubstitute is used to substitute the plus character ('+') in OCI tags.
+// An OCM version is allowed to contain the plus character, but OCI tags do not allow it.
+// Because the OCI tag of an artifact representing an OCM component is derived from the respective component
+// version, this replacement is required.
```

Comment:
I think it would be good if you could reference the relevant part of the OCM spec here, but otherwise this is great 💯